src/storage/src/monitor/monitored_store.rs

```diff
@@ -367,7 +368,7 @@ impl<S: StateStoreIterItemStream> MonitoredStateStoreIter<S> {
     }
 
     fn into_stream(self) -> impl StateStoreIterItemStream {
-        Self::into_stream_inner(self)
+        Self::into_stream_inner(self).instrument(tracing::trace_span!("store_iter"))
```

Comment:
Does this mean that each time of storage iteration will be shown as a span in the Jaegar's graph? If so, it sounds like there will a large amount of data, and it will actually be row-level instead of "epoch-level".

Reply:
> Does this mean that each time of storage iteration will be shown as a span in the Jaegar's graph? If so, it sounds like there will a large amount of data,

Yes but only if the tracing level is set to `TRACE`. Currently, we follow the logging level filter so the levels should be `DEBUG` or `INFO` by default in production.

> it will actually be row-level instead of "epoch-level".

By "epoch-level" I mean the spans are traced in the granularity of "epoch". That is, operations in different epochs are records under different root spans, then different pages in Jaeger.